components/english/vocabulary-practice.tsx

```
@@ -24,7 +24,7 @@ type VocabularyCard = {
   saved: boolean;
 };
 
-export function VocabularyPractice() {
+export function VocabularyPractice({ vocabularies }: { vocabularies: any[] }) {
```

ESLint

[error] 'vocabularies' is defined but never used.

(@typescript-eslint/no-unused-vars)